### DISCUSSION TITLE
Decent basic working example?

### DISCUSSION THREAD

**Discussion by boardmain** [node-oauth/node-oauth2-server#245] [Q&A]
There is a simple working basic example of the server working with express? and updated?

**Comment by jorenvandeweyer** [comment]
Hi there! You can find a basic example [here](https://github.com/node-oauth/express-oauth-server/tree/master).

**Reply by jankapunkt** [reply]
@boardmain does that solve your question? If so, would you mind marking this as answered?

**Comment by binajmen** [comment]
Hi,

Are the examples supposed to work out of the box ? Because it doesn't seem to work for me.

I've tried to implement a minimal example by myself so I can test the whole workflow, but I'm stuck with different pieces of the puzzle :)

Current work in progress:
https://github.com/binajmen/node-oauth2-server/

I'm really struggling to understand how to assemble all the pieces together. It is unclear which routes should be created, to handle which cases. Any help would be greatly appreciated 🙏

**Reply by jankapunkt** [reply]
We currently move the examples into the example repository in this org: https://github.com/node-oauth/node-oauth2-server-examples

However, there is only one example yet and the example from the express repository are all highly out-of-date.

**Reply by binajmen** [reply]
Oh thanks, I was not aware of this repo. I'll go through this example.
